GliaWidgets/View/Call/CallStyle.swift

```diff
@@ -1,23 +1,80 @@
 import UIKit
 
+/// Style of the call view.
 public class CallStyle: EngagementStyle {
+    /// Title for audio call.
     public var audioTitle: String
+
+    /// Title for video call.
     public var videoTitle: String
+
+    /// Back button style in the header.
     public var backButton: HeaderButtonStyle
+
+    /// Close button style in the header.
     public var closeButton: HeaderButtonStyle
+
+    /// A text to display operator's name. Include `{operatorName}` template parameter in the string to display operator's name.
```

Comment:
"A text to display operator's name." - does not sound like a sentence. Would it actually be "The text to display as the operator's name."?

Reply:
I looked into that property because I was confused by it. Turns out, it's not actually used for anything 😬 
Not sure if it's a functional oversight from our side or just some leftover after design changes. Will investigate further.

Reply:
Alright!